adhocracy4/comments_async/static/comments_async/ai_report.jsx

```diff
@@ -32,31 +33,84 @@ export const AiReport = ({ Report, notificationPk, toggleShowAiReport }) => {
     </button>
   )
 
+  const confidenceToPercent = (confidence) => {
+    const percentFormat = new Intl.NumberFormat('default', {
+      style: 'percent',
+      minimumFractionDigits: 0,
+      maximumFractionDigits: 0
+    })
+    return percentFormat.format(confidence)
+  }
+
+  const extractLabelWords = (label) => {
+    const words = report.explanation[label]
+    let result = ''
+    const length = Math.min(words.length, 3)
+    for (let i = 0; i < length; i++) {
+      result += words[i][0]
+      if (i < length - 1) {
+        result += ', '
+      }
+    }
+    return result
+  }
+
+  const renderExplanation = (
+    <ul>
+      {report.label.map((label, index) => (
+        <li key={label[0]}>
+          {label[1].charAt(0).toUpperCase() + label[1].slice(1) + ' '}
+          ({confidenceToPercent(report.confidence[index])}): {' '}
```

Comment:
@hom3mad3 I don't really know if this is the way to add a whitespace, having it all in one line seems annoying, I guess the other option is to wrap them in spans?